tike/ptycho.py

```diff
@@ -269,8 +272,10 @@ def combine_grids(grids, v, h,
     return combined[1:-1, 1:-1, 0]
 
 
-def uncombine_grids(grids_shape, v, h,
-                    combined, combined_corner):
+def uncombine_grids(
```

Comment:
Using "frame" instead of "grid" sounds better, because e.g. we use frames per second as a detector spec, and it seems more intuitive.

Reply:
Maybe it sounds better in a ptychography context, but don't think the current function name is misleading or ambiguous, so it's not worth my time to change all instances of grid to frame and then check that the docs still make sense.

Reply:
Fair enough.